src/shared/renderUtils.jsx

```diff
@@ -1,19 +1,59 @@
 import React from "react";
 
-function renderBody(body){
-	const paragraphs = body.split("\n"); // splits the string at every line-break
-	return paragraphs.map(renderParagraph);
+function renderBody(data, options = {}){ 
+	const paragraphs = data.split("\n"); // splits the string at every line-break
+
+	return paragraphs.map(
+		renderParagraph.bind(this, options)
```

Comment:
quick couple notes here!

1. When we do a multi-line return, we should wrap inside of parenthesis, so this would be:

```
return (
    paragraphs.map(
        renderParagraph.bind(this, options)
    )
);
```

But also, it might be useful to give that bound `renderParagraph` a new name and keep it all single-line rather than multi-line it?  Something like:

```
const paragraphs = data.split("\n");
const customisedRenderParagraph = renderParagraph.bind(this, options);

return paragraphs.map(customisedRenderParagraph);
```